packages/dd-trace/test/datastreams/writer.spec.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+const pkg = require('../../../../package.json')
+const stubRequest = sinon.stub()
+
+const { DataStreamsWriter } = proxyquire(
+  '../src/datastreams/writer', {
+    '../exporters/common/request': stubRequest
+})
+
+require('../setup/tap')
+
+describe('DataStreamWriter unix', () => {
+
+  let writer
+  const unixConfig = {
+    hostname: '',
+    url: new URL('unix:///var/run/datadog/apm.socket'),
+    port: ''
+  }
+
+  it("should construct unix config", () => {
+    writer = new DataStreamsWriter(unixConfig);
+    expect(writer._url).to.equal(unixConfig.url);
+  })
+
+  it("should call 'request' through flush with correct options", () => {
+    writer = new DataStreamsWriter(unixConfig);
+    writer.flush({}):
```

Comment:
Looks like there's a typo here preventing the test file from parsing

Reply:
Thank you !
I'll have a look today 👀

Reply:
should be good now <3